h2o-docs/src/product/downloading.rst

```diff
@@ -1,25 +1,30 @@
+This section describes how to download and install the latest stable version of H2O. These instructions are also available on the `H2O Download page <http://h2o-release.s3.amazonaws.com/h2o/latest_stable.html>`__.  Please first make sure you meet `the requirements to download and use H2O <https://docs.h2o.ai/h2o/latest-stable/h2o-docs/welcome.html#requirements>`__.  Java is a prerequisite for H2O, even if using it from the R or Python packages.
 
-**Note**: To download the nightly bleeding edge release, go to `h2o-release.s3.amazonaws.com/h2o/master/latest.html <https://h2o-release.s3.amazonaws.com/h2o/master/latest.html>`__. Choose the type of installation you want to perform (for example, "Install in Python") by clicking on the tab. 
+.. note::
+	
+	Check our `Downloads page to download the nightly bleeding edge release <h2o-release.s3.amazonaws.com/h2o/master/latest.html>`__. 
 
-Choose your desired method of use below.  Most users will want to use H2O from either `R <http://docs.h2o.ai/h2o/latest-stable/h2o-docs/downloading.html#install-in-r>`__ or `Python <http://docs.h2o.ai/h2o/latest-stable/h2o-docs/downloading.html#install-in-python>`__; however we also include instructions for using H2O's web GUI Flow and Hadoop below.
+	Scroll down to the H2O section and select Nightly Bleeding Edge. Then, choose the type of installation you want to perform (for example, "Install in Python") by clicking on the tab. 
 
+Choose your desired method of use below.  Most users will want to use H2O from either `R <http://docs.h2o.ai/h2o/latest-stable/h2o-docs/downloading.html#install-in-r>`__ or `Python <http://docs.h2o.ai/h2o/latest-stable/h2o-docs/downloading.html#install-in-python>`__. However, we also include instructions for using H2O's web GUI Flow and Hadoop below.
 
-Download and Run from the Command Line
---------------------------------------
 
-If you plan to exclusively use H2O's web GUI, `Flow <http://docs.h2o.ai/h2o/latest-stable/h2o-docs/flow.html>`__, this is the method you should use.  If you plan to use H2O from R or Python, skip to the appropriate sections below.
+Download and run Flow from the command line
+-------------------------------------------
 
-1. Click the ``Download H2O`` button on the `http://h2o-release.s3.amazonaws.com/h2o/latest_stable.html <http://h2o-release.s3.amazonaws.com/h2o/latest_stable.html>`__ page. This downloads a zip file that contains everything you need to get started.
+If you plan to exclusively use H2O's web GUI, `Flow <http://docs.h2o.ai/h2o/latest-stable/h2o-docs/flow.html>`__, this is the method you should use.
+
+1. Go to our `latest stable release page <http://h2o-release.s3.amazonaws.com/h2o/latest_stable.html>`__.
+2. Click the ``Download H2O`` button from the Download and Run tab. This downloads a zip file that contains everything you need to get started.
```

Comment:
security guidelines tab not working.

Reply:
Odd, this one works for me. I also did not update it and it works on the site.